templates/default/nginx/upstream_thin.conf

```diff
@@ -5,8 +5,8 @@
 upstream redmine_thin {
 <% servers = node['redmine']['thin_servers'].to_i %>
 <% (0...servers).each do |i| %>
-	server unix:/var/run/redmine/thin.<%= i %>.sock max_fails=1 fail_timeout=15s;
+	server unix:/var/run/thin/thin.<%= i %>.sock max_fails=1 fail_timeout=15s;
```

Comment:
how about using `redmine` as socket name here? `/var/run/thin/redmine.0.sock` sounds more reasonable to me.